docs/manual/WARRANTY.md

```diff
@@ -0,0 +1,18 @@
+# Warranty and Return information
```

Comment:
Lifted this file wholesale from [https://www.micro-nova.com/amplipi/warranty](our website) with a few edits
Edits include adding a line to the end of the warranty and replacing the link with our actual support email address, and a few minor sentence flow changes